style.css

```diff
@@ -0,0 +1,37 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: black;
+  color: white;
+}
+
+.img-div {
+  justify-items: center;
+  width: 100%;
+  height: auto;
+  align-items: center;
+}
+
+#img-caption {
+  width: 400px;
+}
+
+#image {
+  width: "100%";
+  height: "20%";
+  max-width: 100%;
+  display: block;
+}
```

Comment:
change the image height and width set header image in the heading area.

Reply:
- [ ] As mentioned by the previous code reviewer , let's make it perfect by giving a short explanation for the "why" or reason for the request you are making.